pkgs/tools/security/age/default.nix

```diff
@@ -1,17 +1,39 @@
-{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+{ lib, buildGoModule, fetchFromGitHub, fetchpatch, installShellFiles }:
 
 buildGoModule rec {
   pname = "age";
-  version = "1.0.0";
-  vendorSha256 = "sha256-Hdsd+epcLFLkeHzJ2CUu4ss1qOd0+lTjhfs9MhI5Weg=";
+  version = "1.1.1";
+  vendorSha256 = "sha256-x2+r03A3lFn/VqpiC5/IEWQUNFZbcPQM9wHTYknAck8=";
+  proxyVendor = true;
```

Comment:
Seems it skips some tests but otherwise builds and passes the tests is proxyVendor is unset, this should have a comment so it isn't inadvertently removed.

Reply:
i think messed something up. no tests should be skipped. (i was certain i tested this).
will take a closer look in my morning.

Reply:
Sorry, I didn't look closely enough at the output so my previous comment my be incorrect. It skips downloading some things during the tests, I haven't checked if it is actually skipping tests. 

With proxyVendor
```
running tests
go: downloading c2sp.org/CCTV/age v0.0.0-20221230231406-5ea85644bd03
ok      filippo.io/age  0.379s
ok      filippo.io/age/agessh   0.308s
ok      filippo.io/age/armor    0.002s
go: downloading github.com/FiloSottile/go-internal v1.8.2-0.20220728122003-0ced171a3e0e
go: downloading github.com/pkg/diff v0.0.0-20210226163009-20ebb0f2a09e
go: downloading github.com/creack/pty v1.1.18
ok      filippo.io/age/cmd/age  0.402s
ok      filippo.io/age/internal/bech32  0.002s
ok      filippo.io/age/internal/format  0.001s
ok      filippo.io/age/internal/stream  0.006s
```

Without proxyVendor
```
running tests
ok      filippo.io/age  0.379s
ok      filippo.io/age/agessh   0.198s
ok      filippo.io/age/armor    0.002s
ok      filippo.io/age/cmd/age  0.400s
ok      filippo.io/age/internal/bech32  0.002s
ok      filippo.io/age/internal/format  0.001s
ok      filippo.io/age/internal/stream  0.007s
```

Also you commit message is missing a colon after the package name so CI didn't built it automatically.

Reply:
* Due to use of `//go:embed` we wouldn't need even need `proxyVendor = true`. I forgot it there from a previous iteration. Now removed.

* Verified it runs the same tests.
```
# modified derivation with `preCheck = ''buildFlagsArray+=("-v")'';`
% nix build --allow-dirty -Lv --no-link .#age |& grep '=== RUN' | sed 's/^age> //' > /tmp/age-nix-build.tests

# age repo. current main.
% go test -v ./... |& grep '=== RUN' > /tmp/age-repo-main.tests

% diff -u /tmp/age-{nix-build,repo-main}.tests
%
```

* Added the missing colon to the commit message.